docs/contributing/core/index.md

```diff
@@ -40,7 +40,7 @@ However, the following links and sections below may be helpful.
 ```{include} ../../volto/contributing/install-operating-system.md
 ```
 
--   Python {SUPPORTED_PYTHON_VERSIONS}
+-   Python {SUPPORTED_PYTHON_VERSIONS_PLONE60}
```

Comment:
We should use the Plone 6.1 versions here.
```suggestion
-   Python {SUPPORTED_PYTHON_VERSIONS_PLONE61}
```

Reply:
We support both 6.0 and 6.1. For those stragglers on 5.2, 6.0 is a bridge to modern Python versions. I would like not to leave them behind. Is there a very good reason not to have the following instead?

```md
-   Plone 6.0 requires Python version {SUPPORTED_PYTHON_VERSIONS_PLONE60}.
-   Plone 6.1 requires Python version {SUPPORTED_PYTHON_VERSIONS_PLONE61}.
```

Reply:
I'm fine with that. Though, for the purposes of working on buildout.coredev, it probably makes the most sense to pick one of the Python versions that is compatible with both, so that it's easier to switch back and forth if needed.